Realm/RealmSwift/List.swift

```diff
@@ -39,30 +56,69 @@ public final class List<T: Object>: ListBase, SequenceType {
 
     // MARK: Initializers
 
+    /// Creates a `List` that holds objects of type `T`.
     public override init() {
         super.init(array: RLMArray(objectClassName: T.className()))
     }
 
+    /**
+    Create a `List` by passing in the `RLMArray` to be wrapped.
+
+    :param: rlmArray `RLMArray`.
+    */
```

Comment:
Jazzy supports a minimum ACL setting and we'll be using `public`, so this won't show up in user-facing documentation.